Exadel.Compreface.UnitTests/Services/SubjectExampleServiceTests.cs

```diff
@@ -60,7 +60,7 @@ public async Task AddExampleSubjectAsync_TakesRequestModel_ReturnsProperResponse
     }
 
     [Fact]
-    public async Task AddBase64ExampleSubjectAsync_TakesRequestModel_ReturnsProperResponseModel()
+    public async Task AddBase64SubjectExampleAsync_TakesRequestModel_ReturnsProperResponseModel()
```

Comment:
this test doesn't work

Expected: Exadel.Compreface.DTOs.ExampleSubjectDTOs.AddBase64ExampleSubject.AddBase64ExampleSubjectResponse
Actual:   (null)

Reply:
I did not changed it, someone accidentally commited to my PR

Reply:
I found the problem and fixed it